integration-tests/integration/prisonerProfile/caseNotesPage.spec.js

```diff
@@ -63,10 +63,30 @@ context('A user can view prisoner case notes', () => {
     tableDataRow.createdBy().contains('Tuesday 31/10/2017 01:30 Mickey Mouse')
     tableDataRow
       .caseNoteDetails()
-      .contains('Key Worker: Key Worker Session This is some text Add more details Happened: 31/10/2017 - 01:30')
+      .contains(
+        'Incentive Level: Incentive Level Warning This is some text Add more details Happened: 31/10/2017 - 01:30'
+      )
     tableDataRow
       .caseNoteAddMoreDetailsLink()
       .contains('Add more details')
       .should('have.attr', 'href', 'http://localhost:20200/offenders/A12345/case-notes/12311312/amend-case-note')
+    tableDataRow
+      .caseNotePrintIncentiveLevelSlipLink()
+      .contains('Print Incentive Level Slip')
+      .should(
+        'have.attr',
+        'href',
+        '/iep-slip?offenderNo=A12345&offenderName=Smith%2C%20John&location=HMP%20Moorland&casenoteId=12311312&issuedBy=undefined'
+      )
+
+    tableDataRow.caseNotePrintIncentiveLevelSlipLink().click()
+
+    tableDataRow
+      .caseNotePrintIncentiveLevelSlipLink()
+      .invoke('removeAttr', 'target')
+      .click()
```

Comment:
Cypress can't cope with multiple windows so this temporarily strips the target="_blank" from the link so it loads in the same window and can be interrogated.